packages/commercetools-api-request-builder/src/default-services.js

```diff
@@ -57,6 +155,7 @@ export default {
       features.projection,
     ],
   },
+  // TODO: productSuggestions
```

Comment:
I've added it in the project backlog, which can be converted into an issue

https://github.com/commercetools/nodejs/projects/1